README.md

```diff
@@ -79,6 +79,25 @@ Output artifacts:
 
 You can then use `insertion-tree.qza` for all downstream analyses, e.g. "Alpha and beta diversity analysis", instead of `rooted-tree.qza`.
 
+### Expected Runtimes
+
+We ran ``qiime fragment-insertion sepp`` on four of 32 cores of a Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz server with 265 GB available RAM. Reported "time" is accumulated "user time". Runtime only depends on number of sOTUs, not on number of samples.
+
+| QIITA ID |sOTUs	|samples|sOTU length| memory (max RSS)|time (hh:mm)|wall time (hh:mm)|
+|:--------:|-------:|------:|----------:|----------------:|:----------:|:---------------:|
+| 1024	   |21,473	|344	|150 nt	    |10.2 GB	      |06:32	   |01:54            |
+| 10315	   |31,784	|199	|150 nt	    |10.2 GB	      |09:35	   |02:47            |
+| 10343	   |14,245	|389	|150 nt	    |10.3 GB	      |06:10	   |01:45            |
+| 10346	   |108,447	|1,292	|100 nt	    |10.4 GB	      |20:59	   |06:07            |
+| 10422	   |4,702	|647	|150 nt	    |10.4 GB	      |01:37	   |00:31            |
+| 2014	   |23,029	|1,017	|150 nt	    |10.2 GB	      |08:45	   |02:27            |
+| 2136	   |29,702	|504	|150 nt	    |10.4 GB	      |08:48	   |02:33            |
+| 550	   |27,791	|1,967	|100 nt	    |10.4 GB	      |05:49	   |01:43            |
+| 850	   |11,301	|528	|90 nt	    |10.2 GB	      |02:07	   |00:40            |
+| MrOS	   |4,727	|599	|249 nt	    |10.3 GB	      |02:44	   |00:48            |
```

Comment:
What's MrOS?

Reply:
MrOS is an unpublished study which (at the time of compiling the table) wasn't yet uploaded to Qiita - but I am using this dataset in different parts of the manuscript and the above table is copy and pasted from this manuscript. I now managed to put it into Qiita, but it is not public